thrift/arg2/testutils/arg2_kv_builder.go

```diff
@@ -0,0 +1,23 @@
+package testutils
```

Comment:
let's move this to `testutils/thriftarg2test` or something, `testutils` is too generic of a package and clashes with the other `testutils` we already have